sssd_test_framework/roles/client.py

```diff
@@ -115,3 +115,16 @@ def sss_ssh_knownhosts(self, *args: str) -> ProcessResult:
         :rtype: ProcessResult
         """
         return self.host.conn.exec(["sss_ssh_knownhosts", *args])
+
+    def sss_ssh_authorizedkeys(self, *args: str) -> ProcessResult:
+        """
+        Execute sss_ssh_authorizedkeys.
+
+        :param `*args`: Command arguments.
+        :type `*args`: str
+        :return: Command result.
+        :rtype: ProcessResult
+        """
+        result = self.host.conn.exec(["sss_ssh_authorizedkeys", *args], raise_on_error=False)
+
+        return result
```

Comment:
Couldn't this be combined and drop the result variable?

`return self.host.exec([...])`